formats.js

```diff
@@ -1,6 +1,7 @@
 var moment = require('moment');
 var format = require('util').format;
 var validUrl = require('valid-url');
+const decimals = 3;
```

Comment:
* Maybe a bit more describing name of this? I do not understand the purpose based on this, at least.
* capital letters.